frontend/src/services/logStreamService.ts

```
@@ -0,0 +1,132 @@
+    // Parse circuit events
+    const eventData = message.match(/EVENT:(.*):ENDEVENT/)
+    if (eventData && eventData[1]) {
+      try {
+        const parsedData = JSON.parse(eventData[1])
+        switch (parsedData.event) {
+          case 'CIRCUIT_BUILT':
+            const circuit: Circuit = {
+              id: parsedData.circuit_id,
+              relays: parsedData.relays,
+            }
+            console.log('🔄 LogStreamService: Circuit built:', circuit)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)